### PR TITLE
Stablize python test

### PR DESCRIPTION
**Pull request by GitPaean**
referring to the discussion in https://github.com/OPM/opm-simulators/pull/4855 . 

The current way with the python tests

mpi.python_fluidstate_variables
mpi.python_primary_variables

is very vulnerable to the change of the time step sizes.  

In this PR, it is suggested to use smaller time step for the test so it will remove to some extent the failures related to change of time step sizes. 

With the form in the PR, it might still be vulnerable to changes of the iteration numbers, but it should be much less frequent than the form in the master branch. 

And the changes from this PR should not affect the purpose of the tests.

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this please

**Review by hakonhagland — APPROVED** [review]

File: python/test_data/SPE1CASE1a/SPE1CASE1.DATA [hunk 1]
```diff
@@ -422,7 +422,8 @@ WCONINJE
 
 TSTEP
 --Advance the simulater once a month for TEN years:
-31 28 31 30 31 30 31 31 30 31 30 31 /
+10*1 /
```
Comment (hakonhagland) [inline python/test_data/SPE1CASE1a/SPE1CASE1.DATA:425]: Maybe add a comment why the first time steps are chosen as 10 days?

File: python/test/test_basic.py [hunk 2]
```diff
@@ -54,9 +54,9 @@ def test_all(self):
             sim.step_init()
             sim.step()
             dt = sim.get_dt()
-            # NOTE: The timestep should be 1 month = 31 days
-            #  = 31 * 24 * 60 * 60 seconds = 2678400 seconds
-            self.assertAlmostEqual(dt, 2678400.0, places=7, msg='value of timestep')
+            # NOTE: The timestep should be 1 day
```
[the remark this answers is not in the document]
Comment (GitPaean) [reply python/test/test_basic.py:57]: Will do when back to office.

File: python/test_data/SPE1CASE1a/SPE1CASE1.DATA (hunk 1 above)
Comment (GitPaean) [reply python/test_data/SPE1CASE1a/SPE1CASE1.DATA:425]: It is 10 time steps. Each time step is 1 day.

Comment (GitPaean) [reply python/test_data/SPE1CASE1a/SPE1CASE1.DATA:425]: Will add comments also.

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
With the approval and jenkins green. I am getting this PR in now.

**Pull request merged**